fed/mosaic-sumo/src/main/java/org/eclipse/mosaic/fed/sumo/ambassador/SumoAmbassador.java

```diff
@@ -392,37 +392,46 @@ private void applyChangesInVehicleTypeForVehicle(String vehicleId, VehicleType a
 
     private String extractDepartureSpeed(VehicleRegistration vehicleRegistration) {
         switch (vehicleRegistration.getDeparture().getDepartureSpeedMode()) {
-            case PRECISE:
+            case PRECISE -> {
                 return String.format(Locale.ENGLISH, "%.2f", vehicleRegistration.getDeparture().getDepartureSpeed());
-            case RANDOM:
+            }
+            case RANDOM -> {
                 return "random";
-            case MAXIMUM:
-            default:
+            }
+            default -> {
```

Comment:
I would rather add all cases here explicitly and remove default instead of hiding "MAXIMUM" within the default case. That way the compiler would tell you that this part of the code need handling of a new case if you add one to the departure speed mode.

```java
return switch (vehicleRegistration.getDeparture().getDepartureSpeedMode()) {
    case PRECISE -> String.format(Locale.ENGLISH, "%.2f", vehicleRegistration.getDeparture().getDepartureSpeed());
    case RANDOM -> "random";
    case MAXIMUM -> "max";
};
```